mytoyota/models/lock_status.py

```diff
@@ -0,0 +1,247 @@
+"""Models for vehicle sensors."""
+from __future__ import annotations
+
+from datetime import datetime
+from typing import List, Optional
+
+from mytoyota.models.endpoints.status import (
+    RemoteStatusModel,
+    RemoteStatusResponseModel,
+    SectionModel,
+    VehicleStatusModel,
+)
+
+
+def _get_category(data: Optional[RemoteStatusModel], category: str) -> Optional[VehicleStatusModel]:
+    if data is not None:
+        return next(
+            (item for item in data.vehicle_status if item.category == category),
+            None,
+        )
+    return None
+
+
+def _get_section(data: Optional[VehicleStatusModel], section: str) -> Optional[SectionModel]:
+    if data is not None:
+        return next(
+            (item for item in data.sections if item.section == section),
+            None,
+        )
+    return None
+
+
+def _get_status(data: Optional[SectionModel], status: str) -> Optional[bool]:
+    if data is not None:
+        return not bool(
```

Comment:
It seems that the values here have an inverse logic. 0 means closed (`True`), 1 means open (`False`), etc...